etp-backend/src/main/clj/solita/etp/service/luokittelu.clj

```diff
@@ -0,0 +1,12 @@
+(ns solita.etp.service.luokittelu
+  (:require [clojure.java.jdbc :as jdbc]
+            [solita.etp.db :as db]))
+
+(def luokittelut #{"ilmanvaihto lammitysmuoto lammonjako"})
+
+(defn- select-luokittelu [db table-name]
```

Comment:
Idea on, että select-luokittelu saa minkä tahansa taulun nimen argumentiksi ja kysely rakennetaan stringinä. select-luokittelu on kuitenkin private, joten mitään vahinkoa sillä ei saa aikaan. Nimiavaruudesta ulospäin näkyvät tarkat luokittelukohtaiset funktiot, jotka on määritelty alla.